### PR TITLE
feat: H5端与iOS端，实现先获取焦点，再输入文本的能力，对齐Android端 

### PR DESCRIPTION
**Pull request by caofengbin**
> Whether this PR is eventually merged or not, Sonic will thank you very much for your contribution. 
> 
> 无论此PR最终是否合并，Sonic组织都非常感谢您的贡献。

### Checklist

- [x] The title starts with fix, fea, or doc. | 标题为fix、feat或doc开头。
- [x] I have checked that there are no duplicate pull requests with this request. | 我已检查没有与此请求重复的拉取请求。
- [x] I have considered and confirmed that this submission is valuable to others. | 我已经考虑过，并确认这份呈件对其他人很有价值。
- [x] I accept that this submission may not be used. | 我接受此提交可能不会被使用。

### Description

[Agent端MR](https://github.com/SonicCloudOrg/sonic-agent/pull/354) 配套的前端UI修改

### CODE REVIEW

File: src/components/StepUpdate.vue [hunk 1]
```diff
@@ -1901,7 +1909,7 @@ onMounted(() => {
 
       <div
         v-if="
-          step.stepType === 'sendKeys' || step.stepType === 'webViewSendKeys'
+          step.stepType === 'sendKeys' || step.stepType === 'webViewSendKeys' || step.stepType === 'webViewSendKeysByActions'
         "
```
Comment (ZhouYixun) [inline src/components/StepUpdate.vue:1913]: 这里是不是错了

Comment (ZhouYixun) [reply src/components/StepUpdate.vue:1913]: sendKeysByActions 应该跟这里一样？

Comment (caofengbin) [reply src/components/StepUpdate.vue:1913]: 嗯，native 侧的 sendKeysByActions 那里多了一个 提示alert

```
<el-alert
          show-icon
          style="margin-bottom: 10px"
          close-text="Get!"
          type="info"
        >
          <template #title>
            <div>
              TIPS: 使用Android Driver在Flutter页面输入文本时使用此方式。
            </div>
            <div>需要临时变量或全局变量时，可以添加{{ 变量名 }}的形式。</div>
          </template>
        </el-alert>
```

别的都是一样的。感觉可以复用成一个，是否展示el-alert 用个方法单独判断一下就好了。

**Review by ZhouYixun — APPROVED** [review]
mg

**Pull request merged**